documentation/Logical Volume Manager (LVM) format.asciidoc

```diff
@@ -291,8 +293,7 @@ Contains a list of strings. See section: <<status_flags,Status flags>>
 | flags | The flags +
 Contains a list of strings. See section: <<flags,Flags>>
 | extent_size | The size of an extent +
-The value contains the number of sectors +
-[yellow-background]*Note that the sector size is currently assumed to be 512 bytes.*
+The value contains the number of sectors
```

Comment:
It's not an assumption - the redhat documentation states it's defined as 512 bytes:
[The extent size (in units of sectors which are defined as 512 bytes) ](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/5/html/Logical_Volume_Manager_Administration/metadata_contents.html)

That's why I've added the sector size in the Layout section.